playbooks/roles/gitlab/templates/gitlab.hcl.j2

```diff
@@ -1,5 +1,6 @@
 job "{{ nomad_gitlab_job_name }}" {
   datacenters = ["{{ nomad_datacenter }}"]
+  namespace = "{{ nomad_namespaces[1] }}"
   type = "service"
 
   constraint {
```

Comment:
explain I don't get this... why is in 1? and will there always be a second namespace?
This can be done much more robust.

Reply:
This was an attempt to give the ability to configure the names of the namespaces but it is a bit of hack.
Maybe it would be better to put the namespaces in their own variable? Something like:

```yaml
nomad_admin_namespace: admin
nomad_client_namespace: client
```

This mainly being done to allow the namespace names to be configurable.